src/_posts/databases/mysql/2000-01-01-managing.md

```diff
@@ -312,6 +312,16 @@ You will be able to view and copy it only once.
    User "my_user" created with password "YANs3y07m5_KJC2MSDGebh8tx1lliFWh2Yb239zVqGQvbElWDjIN7QWspVH92Ul8".
    ```
 
+### Updating the Default User Password
+
+To change the default user password, contact [support@scalingo.com](support@scalingo.com) as this can only be done by the Scalingo team.
```

Comment:
suggestion: email is not the only way to contact the support. I think we should either be exhaustive or not list the different ways.
```suggestion
To change the default user password, contact the support as this can only be done by the Scalingo team.
```

Reply:
Making the change, but not this is inconsistent across the rest of the site.
Sometimes we give the email, sometimes we say contact via email or chat.

"Contact the support" doesn't sound correct
"Contact support" or "Contact the support team" sound better IMO